tests/gl/ellipse/main.c

```diff
@@ -0,0 +1,74 @@
+#include "tft8.h"
+#include "board.h"
+#include "gl.h"
+
+/* === APPLICATION CONFIGURATION === */
+
+/*
+ * TFT configuration.
+ */
+
+#define TEST_GL_ELLIPSE_TFT_MAP_PINOUTS TFT8_MAP_PINOUTS_16BIT
+#define TEST_GL_ELLIPSE_TFT_BOARD       TFT_BOARD_4_CAPACITIVE
+
+/*
+ * Color configuration.
+ */
+
+#define TEST_GL_ELLIPSE_COLOR_CLEAR         GL_BLUE
+#define TEST_GL_ELLIPSE_COLOR_PEN           GL_RED
+#define TEST_GL_ELLIPSE_COLOR_BRUSH_STYLE   GL_BRUSH_STYLE_FILL
+#define TEST_GL_ELLIPSE_COLOR_BRUSH         GL_GREEN
+#define TEST_GL_ELLIPSE_COLOR_BRUSH_FROM    GL_DARK_GREEN
+#define TEST_GL_ELLIPSE_COLOR_BRUSH_TO      GL_LIGHT_GREEN
+
+/*
+ * Size configuration - in pixels.
+ * NOTE: You can make a circle by setting the last two macros to the same value.
+ */
+
+#define TEST_GL_ELLIPSE_SIZE_PEN_INNER      5
+#define TEST_GL_ELLIPSE_SIZE_PEN_OUTER      10
+#define TEST_GL_ELLIPSE_SIZE_BRUSH_WIDTH    400 // NOTE: Must be even.
+#define TEST_GL_ELLIPSE_SIZE_BRUSH_HEIGHT   200 // NOTE: Must be even.
+
+/*
+ * Center configuration - in screen percentage.
+ */
+
+#define TEST_GL_ELLIPSE_CENTER_X  0.5f
+#define TEST_GL_ELLIPSE_CENTER_Y  0.5f
+
+/* === APPLICATION CODE === */
+
+static gl_driver_t gl_driver;   // Graphic library driver.
+static tft8_cfg_t tft_cfg;      // TFT configuration.
+
+void main() {
+    // Initialize the GL driver.
+    TEST_GL_ELLIPSE_TFT_MAP_PINOUTS(tft_cfg);
+    tft_cfg.board = &TEST_GL_ELLIPSE_TFT_BOARD;
+    tft8_init(&tft_cfg, &gl_driver);
+    
```

Comment:
I believe there are some spaces left in the code.

Make sure to save all files with removing surpluss spaces.

![image](https://user-images.githubusercontent.com/35097995/213996066-4ceaa29a-03ec-4129-ae19-579a0b13e542.png)

Of course, make sure to do this for all files.